### PR TITLE
Add payload saving to cf-helloworld-client.

### PR DESCRIPTION
**Pull request by boaks**
Add saving the payload for demonstrating a blockwise transfer using the cf-simplefile-server.

Signed-off-by: Achim Kraus <achim.kraus@bosch-si.com>

### CODE REVIEW

File: demo-apps/cf-helloworld-client/src/main/java/org/eclipse/californium/examples/GETClient.java [hunk 1]
```diff
@@ -51,12 +54,20 @@ public static void main(String args[]) {
...
 				System.out.println(response.getOptions());
-				System.out.println(response.getResponseText());
-				
-				System.out.println("\nADVANCED\n");
-				// access advanced API with access to more details through .advanced()
-				System.out.println(Utils.prettyPrint(response));
-				
+				if (args.length > 1) {
+					try (FileOutputStream out = new FileOutputStream(args[1])) {
+						out.write(response.getPayload());
```
Comment (sophokles73) [inline demo-apps/cf-helloworld-client/src/main/java/org/eclipse/californium/examples/GETClient.java]: closing the file?

File: demo-apps/cf-helloworld-client/src/main/java/org/eclipse/californium/examples/GETClient.java [hunk 2]
```diff
@@ -51,12 +54,20 @@ public static void main(String args[]) {
...
+					} catch (IOException e) {
+						e.printStackTrace();
+					}
+				} else {
+					System.out.println(response.getResponseText());
+
+					System.out.println("\nADVANCED\n");
```
Comment (sophokles73) [inline demo-apps/cf-helloworld-client/src/main/java/org/eclipse/californium/examples/GETClient.java]: now that you're on this class anyway, can we replace `\n` with `System.lineseparator`?

File: demo-apps/cf-helloworld-client/src/main/java/org/eclipse/californium/examples/GETClient.java (hunk 1 above)
Comment (boaks) [inline demo-apps/cf-helloworld-client/src/main/java/org/eclipse/californium/examples/GETClient.java]: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

using "try (Closable) {}" will do the `close()`.

File: demo-apps/cf-helloworld-client/src/main/java/org/eclipse/californium/examples/GETClient.java (hunk 2 above)
Comment (boaks) [inline demo-apps/cf-helloworld-client/src/main/java/org/eclipse/californium/examples/GETClient.java]: sure.

**Comment by boaks** [comment]
I replaced the '\n' in Utils also.

File: demo-apps/cf-helloworld-client/src/main/java/org/eclipse/californium/examples/GETClient.java (hunk 1 above)
Comment (sophokles73) [inline demo-apps/cf-helloworld-client/src/main/java/org/eclipse/californium/examples/GETClient.java]: oh, sure. I totally missed that in the code ...

**Pull request merged**